src/DblParser/dune

```diff
@@ -1,5 +1,10 @@
-(ocamlyacc (modules YaccParser))
 (ocamllex  (modules Lexer))
+;(ocamlyacc (modules YaccParser));
```

Comment:
Should we remove this line instead of commenting it out? As discussed on the last meeting, we are going to use nice features of cpspg (which are not implemented yet), so I think in the near future, we will be incompatible with ocamlyacc.